package.json

```diff
@@ -71,6 +71,7 @@
     "redux-thunk": "^2.3.0",
     "reflect-metadata": "^0.1.13",
     "reselect": "^4.0.0",
+    "sanitize-html": "^2.3.2",
```

Comment:
I see that we have a lot of ^ but it seems to be a mistake. We must update our dependencies manually, after making sure that's OK with CQs.